server/entity/src/mob.rs

```diff
@@ -132,3 +133,97 @@ pub fn spawn_packet_creator(kind: MobKind) -> SpawnPacketCreator {
+        "minecraft:hopper_minecart" => None,
+        "minecraft:horse" => Some(passive::horse::create),
+        "minecraft:husk" => Some(hostile::husk::create),
+        "minecraft:illusioner" => None,
+        "minecraft:iron_golem" => None,
+        "minecraft:item_frame" => None,
+        "minecraft:leash_knot" => None,
+        "minecraft:lightning_bolt" => None,
+        "minecraft:llama" => Some(neutral::llama::create),
+        "minecraft:llama_spit" => None,
+        "minecraft:mooshroom" => Some(passive::mooshroom::create),
+        "minecraft:mule" => Some(passive::mule::create),
+        "minecraft:ocelot" => Some(passive::ocelot::create),
+        "minecraft:panting" => None,
+        "minecraft:parrot" => Some(passive::parrot::create),
+        "minecraft:phantom" => Some(hostile::phantom::create),
+        "minecraft:pig" => Some(passive::pig::create),
+        "minecraft:polar_bear" => Some(neutral::polar_bear::create),
+        "minecraft:potion" => None,
+        "minecraft:puffer_fish" => Some(defensive::pufferfish::create),
+        "minecraft:rabbit" => Some(passive::rabbit::create),
+        "minecraft:salmon" => Some(passive::salmon::create),
+        "minecraft:sheep" => Some(passive::sheep::create),
+        "minecraft:shulker" => Some(hostile::shulker::create),
+        "minecraft:shulker_bullet" => None,
+        "minecraft:silverfish" => Some(hostile::silverfish::create),
+        "minecraft:skeleton" => Some(hostile::skeleton::create),
+        "minecraft:skeleton_horse" => Some(passive::skeleton_horse::create),
+        "minecraft:slime" => Some(hostile::slime::create),
+        "minecraft:small_fireball" => None,
+        "minecraft:snow_golem" => None,
+        "minecraft:snowball" => None,
+        "minecraft:spawner_minecart" => None,
+        "minecraft:spectral_arrow" => None,
+        "minecraft:spider" => Some(neutral::spider::create),
+        "minecraft:squid" => Some(passive::squid::create),
+        "minecraft:stray" => None,
+        "minecraft:tnt" => None,
+        "minecraft:tnt_minecart" => None,
+        "minecraft:trident" => None,
+        "minecraft:tropical_fish" => Some(passive::tropical_fish::create),
+        "minecraft:turtle" => Some(passive::turtle::create),
+        "minecraft:vex" => Some(hostile::vex::create),
+        "minecraft:villager" => Some(passive::villager::create),
+        "minecraft:vindicator" => Some(hostile::vindicator::create),
+        "minecraft:witch" => Some(hostile::witch::create),
+        "minecraft:wither" => Some(boss::wither::create),
+        "minecraft:wither_skeleton" => Some(hostile::wither_skeleton::create),
+        "minecraft:wither_skull" => None,
+        "minecraft:zombie" => Some(hostile::zombie::create),
+        "minecraft:zombie_horse" => None,
+        "minecraft:zombie_pigman" => Some(neutral::zombie_pigman::create),
+        "minecraft:zombie_villigar" => Some(hostile::zombie_villager::create),
```

Comment:
villigar -> villager

Reply:
wow how did I not notice that. I changed the spelling.